### ISSUE TITLE
Panic related to high eta

### ISSUE THREAD

**Issue by alexheretic** [console-rs/indicatif#561]
```
⠁ 00:07:20 ###############################------------------------------------------------------------------------------------------- (sampling crf 23, eta 576413653330y)
thread 'main' panicked at 'overflow when adding durations', library/core/src/time.rs:920:31
stack backtrace:
   0:     0x5579ea415b6c - std::backtrace_rs::backtrace::libunwind::trace::h9a6b80bbf328ba5d
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1:     0x5579ea415b6c - std::backtrace_rs::backtrace::trace_unsynchronized::hd162ec543a11886b
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x5579ea415b6c - std::sys_common::backtrace::_print_fmt::h78a5099be12f51a6
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/sys_common/backtrace.rs:65:5
   3:     0x5579ea415b6c - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::ha1c5390454d74f71
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x5579ea3725bf - core::fmt::write::h9ffde816c577717b
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/fmt/mod.rs:1254:17
   5:     0x5579ea3e68f4 - std::io::Write::write_fmt::h88186074961638e4
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/io/mod.rs:1698:15
   6:     0x5579ea4173df - std::sys_common::backtrace::_print::h184198273ed08d59
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/sys_common/backtrace.rs:47:5
   7:     0x5579ea4173df - std::sys_common::backtrace::print::h1b4d8e7add699453
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/sys_common/backtrace.rs:34:9
   8:     0x5579ea416fde - std::panicking::default_hook::{{closure}}::h393bcea75423915a
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/panicking.rs:269:22
   9:     0x5579ea418040 - std::panicking::default_hook::h48c64f31d8b3fd03
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/panicking.rs:288:9
  10:     0x5579ea418040 - std::panicking::rust_panic_with_hook::hafdc493a79370062
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/panicking.rs:691:13
  11:     0x5579ea417b24 - std::panicking::begin_panic_handler::{{closure}}::h0a64bc82e36bedc7
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/panicking.rs:582:13
  12:     0x5579ea417a86 - std::sys_common::backtrace::__rust_end_short_backtrace::hc203444fb7416a16
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/sys_common/backtrace.rs:150:18
  13:     0x5579ea417a71 - rust_begin_unwind
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/std/src/panicking.rs:578:5
  14:     0x5579ea370672 - core::panicking::panic_fmt::h0f6ef0178afce4f2
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/panicking.rs:67:14
  15:     0x5579ea376f32 - core::panicking::panic_display::h01701c1a3a28d8fa
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/panicking.rs:150:5
  16:     0x5579ea376f32 - core::panicking::panic_str::h7d8fffbb2324dc68
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/panicking.rs:134:5
  17:     0x5579ea376f32 - core::option::expect_failed::h18c286e2b6fe8852
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/option.rs:2025:5
  18:     0x5579ea37a805 - core::option::Option<T>::expect::hff164092e7deb318
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/option.rs:913:21
  19:     0x5579ea37a805 - <core::time::Duration as core::ops::arith::Add>::add::h3e0b64f74ec2b515
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/time.rs:920:31
  20:     0x5579ea38c92f - <indicatif::format::HumanDuration as core::fmt::Display>::fmt::he910c5d8f02111aa
  21:     0x5579ea372553 - core::fmt::run::h3ce546d9a0c94c92
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/fmt/mod.rs:1302:5
  22:     0x5579ea372553 - core::fmt::write::h9ffde816c577717b
                               at /rustc/90c541806f23a127002de5b4038be731ba1458ca/library/core/src/fmt/mod.rs:1270:26
  23:     0x5579ea383efe - indicatif::style::ProgressStyle::format_state::hcf501f60d5460867
  24:     0x5579ea38a0db - indicatif::state::BarState::draw::he2889faf730461f3
  25:     0x5579ea38a5dd - indicatif::state::BarState::update_estimate_and_draw::h9cb3edc42cd4b9df
  26:     0x5579ea38c2b4 - indicatif::progress_bar::ProgressBar::set_position::had2aca3070d0d32d
  27:     0x5579ea2e1a20 - ab_av1::command::crf_search::crf_search::{{closure}}::hf888132e7dc914e4
  28:     0x5579ea313dda - <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll::hb721cac5448a9453
  29:     0x5579ea3023bc - tokio::runtime::context::runtime::enter_runtime::h1a7362b249b9a1df
  30:     0x5579ea3245be - ab_av1::main::hb8fec8b859ad0c86
  31:     0x5579ea300400 - std::sys_common::backtrace::__rust_begin_short_backtrace::h6bf3ec6018ded75e
  32:     0x5579ea323b31 - main
  33:     0x7fb8e9395850 - <unknown>
  34:     0x7fb8e939590a - __libc_start_main
  35:     0x5579ea2b95a5 - _start
  36:                0x0 - <unknown>
```

Downstream https://github.com/alexheretic/ab-av1/issues/146

**Comment by alexheretic** [comment]
Actually though the backtrace mislead me a bit, it seems #560 does indeed fix this.

**Comment by alexheretic** [comment]
Actually possibly this particular trace was already fixed by 1bf2cce7c455f91e7535362be02e228ceb74e4f1.

In which case make a new release would be enough to close this issue. #560 still seems worthwhile anyway.
